src/cnd.h

```diff
@@ -0,0 +1,19 @@
+#ifndef RLANG_CND_H
+#define RLANG_CND_H
+
+#include <stdbool.h>
+
+
+void r_inform(const char* fmt, ...);
+void r_warn(const char* fmt, ...);
+void r_abort(const char* fmt, ...);
+
+SEXP interp_str(const char* fmt, ...);
```

Comment:
yes but I'm writing it as if it was an exported API. I guess you're right that public C functions need a consistent prefix. That will make the API much less friendly to use.